gpt-oss-on-intel-xeon.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Running GPT OSS on Intel Xeon CPU"
+thumbnail: /blog/assets/xxx.png # waiting for upload
+authors:
+- user: Jiqing
+  guest: true
+  org: Intel
+- user: MatrixYao
+  guest: true
+  org: Intel
+- user: kding1
+  guest: true
+  org: Intel
+---
+
+
+# Running GPT-OSS on Intel Xeon CPUs
+
+GPT-OSS achieves inference speeds that approach human reading speed, thanks to its expert computation method on Intel Xeon CPUs. Unlike GPUs, where all experts are computed in parallel and masking is applied to select active values, CPUs do not have extra FLOPs to compute all experts simultaneously. Instead, GPT-OSS on CPUs computes one expert at a time, avoiding unnecessary calculations for inactive experts. This optimization results in a 3x speed-up compared to computing all experts, making it highly efficient for CPU-based inference.
+
+
+## Introduction
+
+GPT-OSS is an open-weight model known for its strong reasoning and versatility. Its MoE architecture, while having a large number of parameters, activates only a small subset during inference. This makes it efficient to run on Intel Xeon CPUs, where Expert Parallelism can further improve performance by distributing the computation of experts across multiple processes.
```

Comment:
"This makes it efficient to run on Intel Xeon CPUs" seem jumps out from nowhere.

Reply:
Changed to `This makes it possible to run large models on Intel Xeon CPUs`